.modernizrrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "minify": true,
+  "options": [
+    "setClasses"
+  ],
+  "feature-detects": [
+    "test/webgl",
+    "test/css/flexbox"
```

Comment:
Word, +1 either way then, we could keep it in for the potential future or drop it, no preference.